microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/get-started.md

```diff
@@ -0,0 +1,366 @@
+
+   **Edit CSS:**
+
+1. In the **Edge DevTools** tab, click the **Select an element in the page to inspect it** (![Inspect tool icon](./open-devtools-and-embedded-browser-images/inspect-tool-icon.png)) button, sometimes called the **Inspect** button.
+
+1. In the **Edge DevTools: Browser** tab, hover over different parts of the page, while you watch the **Elements** tool's DOM tree expand and update.
+
+1. Click the **Success!** heading, which is an `<h2>` element.
+
+1. In the **Styles** tab of the **Elements** tool in the **Edge DevTools** tab, in the non-italic h2 declaration section, click to the right of the `margin-bottom` rule.
+
+1. Enter a new CSS rule, `font-size: 5em`, and then press `Enter`.  The spelling is similar to the rule shown in the italic h2 section from the user agent stylesheet below it.
+
+   `index.html` opens, automatically edited by CSS mirror editing to add the line `font-size: 5em;`.  Scroll to that line, in the h2 section.
+
+  **Arrange tabs:**
+
+1. In the **Edge DevTools** tab, in the non-italic h2 section, click the `font-size` CSS rule that you entered, and then press `Up Arrow` and `Down Arrow`.  The value in `index.html` is automatically edited, by CSS mirror editing.
+
+1. In the **Edge DevTools** tab, click the **Toggle screencast** button a couple times.  The **Edge DevTools: Browser** tab opens and closes, saving space.
+
+1. In the **Edge DevTools: Browser** tab, click the **Close DevTools** or **Open DevTools** button a couple times.  The other **DevTools** tab opens and closes, saving space.
+
+1. Along the top of Visual Studio Code, right-click the tabs and arrange them to show the various tabs at the same time: 
+
+   *  The Side Bar, showing **Microsoft Edge Tools** > **Targets**.
+   *  The `index.html` editor.
+   *  The **Edge DevTools** tab.
+   *  The **Edge DevTools: Browser** tab.
+
+   ![Success page editing h2 CSS size](./open-devtools-and-embedded-browser-images/success-page-edit-h2-css.png)
+
+   **View JavaScript:**
+
+1. In the **Edge DevTools** tab, click the **Console** tool's tab.  `index.html` contains a `<script>` element that contains a JavaScript statement `console.info('Hello from the startpage!')`, which outputs "Hello from the startpage!"
+
+   **Close DevTools:**
+
+1. Click **Close** (**x**) in the **Edge DevTools** tab and in the **Edge DevTools: Browser** tab, if these tabs are open.
+
+*  Select Activity Bar > **Microsoft Edge Tools**.  If the **Targets** section lists any targets, hover on the right side of the target instance and then click **Close instance** (**x**).  The **Launch Instance** button appears, indicating that all DevTools instances are closed.
```

Comment:
This list item uses `*` instead of `1`, which makes it appear as a bullet list item rather than a numbered list item.

Reply:
changed to numbered